src/signals/incident/services/map-paths/index.js

```diff
@@ -0,0 +1,55 @@
+import forEach from 'lodash.foreach';
+import set from 'lodash.set';
+
+import getControls from '../get-controls';
+
+const convertValue = (value) => {
+  if (value === 0) {
+    return 0;
+  }
+  if (value === true) {
+    return 'ja';
+  }
+  if (value === false) {
+    return 'nee';
+  }
+
+  return value;
+};
+
+const mapPaths = (params, incident, wizard) => {
+  forEach(wizard, (step) => {
+    const category_url = incident && incident.subcategory_link ? new URL(incident.subcategory_link).pathname : '';
+    const controls = getControls(step, incident);
+    let mapMerge = {};
+
+    forEach(controls, (control, name) => {
+      const value = incident[name];
+      const meta = control.meta;
+
+      if (meta && meta.isVisible && meta.pathMerge) {
+        const answer = convertValue(value);
+        if (answer || answer === 0) {
+          mapMerge = {
```

Comment:
This is complicated enough to extract to own function along with dedicated tests

Reply:
er wordt eerst een object gemaakt met alle waardes erin en daarna worden die vars in de extra_properties gestopt, maar hoe kan ik dat eenvoudiger maken?